plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/dpdk/DpdkDriverImpl.java

```diff
@@ -70,9 +70,15 @@ public void addDpdkPort(String bridgeName, String port, String vlan, DpdkHelper.
                 dpdkPortVhostUserClientType;
 
         StringBuilder stringBuilder = new StringBuilder();
-        stringBuilder.append(String.format("ovs-vsctl add-port %s %s " +
-                "vlan_mode=access tag=%s " +
-                "-- set Interface %s type=%s", bridgeName, port, vlan, port, type));
+        
+        if (Integer.parseInt(vlan) > 0 && Integer.parseInt(vlan) < 4095) {
+            stringBuilder.append(String.format("ovs-vsctl add-port %s %s " +
+                    "vlan_mode=access tag=%s " +
```

Comment:
could you put only this vland_mode and tag in the if-block and the rest around it. The other parts of the string do not depend on the condition and there is nothing that should go in the else-block.

Reply:
updated